src/main/java/com/mastery/java/task/exceptions/EmployeeExceptionHandler.java

```diff
@@ -0,0 +1,19 @@
+package com.mastery.java.task.exceptions;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class EmployeeExceptionHandler{
```

Comment:
what this class doing in the exceptions package?

Reply:
I took it to the root

Reply:
I may suggest you to move this class to the controllers package.

Reply:
moved it to the controllers